views/aggregate-flow-metadata.json

```diff
@@ -240,13 +254,18 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-census",
-            "description": {
-                "summary": "A flow for triggering a Census sync run and waiting for completion.",
-                "returns": "The result of the sync run."
-            }
+            "path_containing_flow": "prefect_census/flows.py",
+            "repo_url": "https://github.com/PrefectHQ/prefect-census",
+            "slug": "run_census_sync"
         },
         "Wait for Census sync run": {
+            "description": {
+                "summary": "Waits for the Census sync run to finish running."
+            },
+            "documentation_url": "https://prefecthq.github.io/prefect-census/syncs/#prefect_census.syncs.wait_census_sync_completion",
+            "entrypoint": "prefect_census/syncs.py:wait_census_sync_completion",
+            "install_command": "pip install prefect-census",
+            "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/3oznRx2UFkd2XyqNkEZpzB/4e0967a828aec5e2527cedadf8d24e8a/llmjpn8a0pgu8szjmnyi.webp?h=250",
             "name": "Wait for Census sync run",
```

Comment:
Wondering if this should be renamed to "title"

Reply:
i think name is fine, it is literally the `name` attr on the flow

Reply:
The parameters use title:
```
                            "v1_job": {
                                "title": "Job Manifest",
                                "description": "The Kubernetes job manifest to run. This dictionary can be produced using `yaml.safe_load`.",
                                "type": "object"
                            },
```

Reply:
that's what you get if you cast a flow parameter `ParameterSchema` to a dict, I chose the `name` key for the flow summary because it is literally the name of the flow